test/e2e/template_test.go

```diff
@@ -276,6 +275,8 @@ func checkChangesOutput(t *testing.T, actualOutput, expectedOutput string) {
 	// (metadata.managedFields was added and threw off all lines numbers)
 	diffLinesRegexp := regexp.MustCompile(`(?m:^\s*(\d{1,3}\s*|\d{1,3},\s*\d{1,3}|\d{1,3}) [\-+ ])`)
 	actualOutput = diffLinesRegexp.ReplaceAllString(actualOutput, "-linesss-")
+	diffLinesRegexp = regexp.MustCompile(` *\d{1,3}, *\d{1,3}`)
```

Comment:
What prompted us to make this change?

Reply:
This was made to cover `-liness-` from above.

Reply:
The empty line you mean?

Reply:
The empty line with just the line number.

Reply:
Makes sense!